tests/modeladmin/tests.py

```diff
@@ -615,6 +617,49 @@ def test_log_actions(self):
         self.assertEqual(ma.log_change(mock_request, self.band, 'changed'), LogEntry.objects.latest('id'))
         self.assertEqual(ma.log_change(mock_request, self.band, 'deleted'), LogEntry.objects.latest('id'))
 
+    @mock.patch('django.contrib.admin.options.get_deleted_objects')
```

Comment:
Any particular reason why you mock this method? Is that because you want to test the correct method is called? Why don't you just remove real objects from the database?

I think I like your thinking here, but want to be sure that this is what you had in mind :smile: